### PR TITLE
Fixing Broken Links

### PR DESCRIPTION
**Pull request by lyzalou**
@srveit or @jim-phillips 
another for:
https://ctliotest.atlassian.net/browse/CTLIO-418

### CODE REVIEW

File: Backup/automating-sbs-on-api-v2.md [hunk 1]
```diff
@@ -319,4 +319,4 @@ forEach ($Server in $serverDetails)
 
 ### Want to Know More About the Simple Backup Service?
 
-Check out our SBS product page [here](//www.ctl.io/simple-backup-service/). The CenturyLink Cloud [Knowledge Base](../Backup) also has a number of articles on backup, including [Getting Started with Simple Backup](../backup/getting-started-with-simple-backup/) and [Simple Backup How It Works](../backup/simple-backup-service-how-it-works/).
```
Comment (srveit) [inline Backup/automating-sbs-on-api-v2.md]: Change links to `./getting-started-with-simple-backup.md` and `./simple-backup-service-how-it-works.md` respectively.

Comment (lyzalou) [inline Backup/automating-sbs-on-api-v2.md]: I fixed this with # 1549... but you closed it... i'm totally confused now.

Comment (lyzalou) [inline Backup/automating-sbs-on-api-v2.md]: @srveit 🔼

Comment (srveit) [inline Backup/automating-sbs-on-api-v2.md]: You need to add `.md` at the end of the links.

**Pull request merged**